docs/vm/nic-hotplug.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 13
+sidebar_label: NIC Hotplug
+title: "NIC Hotplug"
+keywords:
+  - Harvester
+  - harvester
+  - Virtual Machine
+  - virtual machine
+  - Hotplug
+  - NIC
+description: Hotplug and hotunplug network interfaces to a running VM.
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.6/vm/nic-hotplug"/>
+</head>
+
+_Available as of v1.7.0_
+
+Harvester supports hotplugging and hotunplugging of network interface controllers (NICs) for [live-migratable](./live-migration.md#live-migratable-virtual-machines) virtual machines. This feature allows you to add and remove network interfaces from a running virtual machine without requiring a reboot.
+
+:::note
+
+You cannot perform NIC hotplugging on virtual machines created in clusters running Harvester versions earlier than v1.7.0 _if the network interface's MAC address was not explicitly configured_. To enable this feature, you must restart those virtual machines. Restarting allows the system to automatically save the observed MAC addresses to the virtual machine's configuration.
+
+In addition, virtual machines created by the [Harvester Node Driver](../rancher/node/node-driver.md) do not support NIC hotplugging and hotunplugging.
```

Comment:
Does the Harvester Terraform provider support this? If not, we may also mention it.

Reply:
Good question. I think it's not supported.